doc/en/source/scorpioIntegration.rst

```diff
@@ -149,34 +145,31 @@ for example, generate an alert message when the speed of vehile is greater than
 
 
 How to Push the Generated Result back to the NGSI-LD broker 
-=========================================================================
-
-please register fogfunction2 by default, including its operator, docker image, and fog function. 
-Put them into the initialization list of the designer. 
-
+=============================================================
 
-Step 5: send a update message to trigger fogfunction2
+Step 1: issue a subscription request to the fogflow broker to publish result back to the NGSILD broker
 
 
-.. code-block:: console    
-
-	#please write the curl message to trigger fogfunction2
-
-
-Step 6: check if fogfunction2 is created
-
-
-explain where the users can check if the fogfunction2 is triggered. 
-
-
-Step 7: check if Orion-LD has received the forwarded results
-
+.. code-block:: console
 
-.. code-block:: console    
+        curl -iX POST \
+                  'http://localhost:8070/ngsi-ld/v1/subscriptions' \
+                  -H 'Content-Type: application/json' \
+		  -H 'Integration: true' \
+                  -H 'Accept: application/ld+json' \
+                  -H 'Link: <https://fiware.github.io/data-models/context.jsonld>; rel="https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"; type="application/ld+json"' \
+                  -d ' {
+                        "type": "Subscription",
+                        "entities": [{
+                               "type": "result"
+                        }],
+                      "notification": {
+                          "format": "normalized",
+                          "endpoint": {
+                                   "uri": "http://localhost:1026",
+                                   "accept": "application/ld+json"
+                           }
+                       }
+                   }'
 
```

Comment:
below line should be added in step 6
Fog Function do some data analytics in step no. 4 and publish the analytics result on fogflow broker. NGSILD broker  subscribes fogFlow broker for getting the analytics result.